lambda-functions/elb-cleanup/aws.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/pkg/errors"
+)
+
+// awsTimeout used for the context of AWS SDK
+const awsTimeout = 50 * time.Second
+
+// Client for making AWS requests
+type Client struct {
+	// ec2 *ec2.EC2
+	elbv2 *elbv2.ELBV2
+	elb   *elb.ELB
+}
+
+// Resourcer the interface for the AWS client
+type Resourcer interface {
+	ListUnusedElbs(context context.Context) ([]elbv2.LoadBalancer, error)
```

Comment:
it's more readable and actually valid for go `ListUnusedELB` -> the plural is not needed as we already mentiong with `List` and that we expect many. Same for there rest